src/search/utils/__tests__/QueryStringGenerator.spec.ts

```diff
@@ -1,12 +1,22 @@
 import createQueryString from '../QueryStringGenerator';
-import testData from '../__fixtures__/clauseQueryTestData';
+import {
+  errorFreeTestData,
+  errorThrowingTestData,
+} from '../__fixtures__/clauseQueryTestData';
 
 describe('QueryHelper', () => {
-  testData.forEach(({ description, queryString, clauses }) => {
+  errorFreeTestData.forEach(({ description, queryString, clauses }) => {
```

Comment:
Splitting hairs here, but I would call `errorFreeTestData` `testData` and `errorThrowingTestData` `erroneousTestData`

Reply:
Yeah, I agree on the `testData`. I'm hesitant about `erroneousTestData` in case a developer thinks the test data isn't right. How about `exceptionThrowingTestData` ?